cds_ils/vocabularies/data/alternative_identifier_schemes.json

```diff
@@ -2,7 +2,8 @@
   {
     "type": "alternative_identifier_scheme",
     "key": "ARXIV",
-    "text": "arXiv"
+    "text": "arXiv",
+    "data": { "url_path":  "https://arxiv.org/abs/{identifier}"}
```

Comment:
it was placed initially there, however I was concerned about the data dependency which you mention below, in case of differently defined vocabularies, the component dependent on this field will have issues

Reply:
The UI should not depend on this. The UI should ideally check if, when rendering the vocab., the `data.url_path` exist and render it.
I am honestly not very convinced by fetching the URL from the vocab definition, too many REST calls.